src/lib/util/GameIntegration/FFXIVUtils.ts

```diff
@@ -44,10 +115,14 @@ export async function getCharacterDetails(i18n: Language, id: number) {
 	}
 }
 
-export async function searchCharacter(i18n: Language, name: string) {
+export async function searchCharacter(i18n: Language, name: string, server?: FFXIV.Server) {
 	try {
 		const url = new URL(`${FFXIV_BASE_URL}/character/search`);
 		url.searchParams.append('name', name);
+		if (server) {
+			if (FFXIVServers.includes(server)) url.searchParams.append('server', server);
```

Comment:
```suggestion
			if (FFXIVServers.includes(server.toLowerCase())) url.searchParams.append('server', toTitleCase(server));
```

import toTitleCase from `@klasa/utils`